### PR TITLE
feat: create review list view

### LINKED ISSUE #237: feat: create a user friendly list view of the column review page
Based on user feedback we would like to show the column metadata as a list view in addition to the table view already created.

### PR DESCRIPTION
**Pull request by K-Beicher**
## Description

- This PR adds the feature column review in list view.
- These changes are done because of user feedback and because Kris just needed to see if it could be done, and ended up doing it.
- 
## Related Issues

Closes #237 

## Testing

- [ ] Yes
- [ ] No, not needed (give a reason below)
- [x] No, I need help writing them

I was thinking that I would be able to re-use some of the tests from the previous page, but by 

## Reviewer Focus

This PR requires an in-depth review.

## Checklist

For all PRs that are not general documentation

- [ ] Tests accompany or reflect changes to the code
- [ ] Tests ran and passed locally
- [ ] Ran the linter and formatter
- [ ] Build has passed locally
- [ ] Relevant documentation has been updated

### CODE REVIEW

**Comment by K-Beicher** [comment]
@lwjohnst86, @philter87 am I right in thinking that because this is branched off another branch it won't run tests and doesn't require reviews until the first branch is merged in?

**Comment by lwjohnst86** [comment]
@K-Beicher that's correct 😁

File: sprout/urls.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
         views.column_review,
         name="column-review",
     ),
+    path(
+        "column-review-list/<int:table_id>/",
+        views.column_review_list,
+        name="column-review-list",
```
Comment (philter87) [inline sprout/urls.py:19]: I think ")" is missing in `path(...)`. Maybe you forgot to commit some code

**Review by philter87 — CHANGES_REQUESTED** [review]
Nice!

I just added a picture for the next one reviewing:
image

**Review by philter87 — APPROVED** [review]

**Pull request merged**